fdbserver/BlobWorker.actor.cpp

```diff
@@ -372,6 +373,9 @@ ACTOR Future<BlobGranuleCipherKeysCtx> getLatestGranuleCipherKeys(Reference<Blob
 	cipherKeysCtx.ivRef = makeString(AES_256_IV_LENGTH, *arena);
 	deterministicRandom()->randomBytes(mutateString(cipherKeysCtx.ivRef), AES_256_IV_LENGTH);
 
+	++bwData->stats.getLatestEncryptCipherKeysReqs;
+	bwData->stats.getLatestEncryptCipherKeysMS += int((now() - startTime) * 1000);
```

Comment:
using `now()` with second precision doesn't sound correct; the datapoint would be ZERO most of the time (even when Key is fetched from external KMS) ?

Reply:
Sounds like now() return as `double` but precision is much higher.